config/i18n-tasks.yml

```diff
@@ -122,6 +122,7 @@ ignore_unused:
   - 'moderation.proposals.index.order*'
   - 'moderation.debates.index.filter*'
   - 'moderation.debates.index.order*'
+  - 'moderation.meetings.index.filter*'
```

Comment:
Because those translation keys are dynamic like this: `t("#{i18n_namespace}.filters.#{filter}")` and i18n-tasks complain about that.